README.md

```diff
@@ -1 +1,39 @@
-#TODO
+# vl530lx-ROS-driver
+
+## Install directly
+
+### Build from Source
+
+1. [Install ROS Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu) and the following build tools.
+
+        sudo apt-get install python-wstool python-catkin-tools 
+	
+1. Re-use or create a catkin workspace:
+
+        export CATKIN_WS=~/ws_catkin
+        mkdir -p $CATKIN_WS/src
+        cd $CATKIN_WS/src
+
+1. Download the repository:
+
+        git clone git@github.com:rapyuta-robotics/vl53l0x_ros_driver.git
```

Comment:
clone it recursively, since we have submodule

Reply:
ok